nengo_gui/static/menu.js

```diff
@@ -27,6 +27,8 @@ Nengo.Menu.prototype.show = function (x, y, items) {
         return;
     }
 
+    this.menu_margin = 15;
+
```

Comment:
> In firefox the menu will overflow the bottom out of the window, and go up against the edge for the right hand side

What happens in firefox if the window is fullscreen so it can't overflow out the bottom?  (indeed, all of these checks should be in situations where the window is fullscreen so there's no option for the right-click menu to overflow)

And in any case, from what you're saying, out of the 4 browsers looked at, 3 go right up against the edge and 1 doesn't.  We should look at a few more examples to get a consensus, but the majority seems to be to go right to the edge.